collector/src/descriptor/kind/bridge_server_descriptor.rs

```diff
@@ -111,6 +250,28 @@ fn take_uniq<'a>(
     }
 }
 
+fn take_multi_descriptor_lines<'a>(
+    map: &'a mut HashMap<&str, Vec<DescriptorLine>>,
+    key: &str,
+    len: usize,
+) -> Result<Vec<DescriptorLine<'a>>, Error> {
+    if let Some(v) = map.remove(key) {
+        let format_ok = v.iter().fold(true, |acc, e| {
+            if e.values.len() < len {
+                false
+            } else {
+                acc
+            }
+        });
+        if v.len() < len || !format_ok {
```

Comment:
I'm not sure why checking `v.len() < len`? It means if we want lines with, say 5 arguments, we also need to have at least 5 such lines